packages/commonwealth/client/scripts/views/pages/CommunityManagement/CommunityProfile/Form/Form.tsx

```diff
@@ -0,0 +1,411 @@
+              disabled={nameFieldDisabledState.isDisabled}
+              onInput={(e) => {
+                setNameFieldDisabledState((prevVal) => ({
+                  ...prevVal,
+                  canDisable:
+                    e?.target?.value?.trim() === community.name.trim(),
+                }));
+              }}
+              iconRight={
+                <CWIcon
+                  weight="fill"
+                  className="lock-icon"
+                  iconName={
+                    nameFieldDisabledState.isDisabled
+                      ? 'lockedNew'
+                      : 'lockOpenNew'
+                  }
+                  disabled={!nameFieldDisabledState.canDisable}
+                  onClick={() => {
+                    if (nameFieldDisabledState.canDisable) {
+                      setNameFieldDisabledState((prevVal) => ({
+                        ...prevVal,
+                        isDisabled: !prevVal.isDisabled,
+                      }));
+                    }
+                  }}
+                />
+              }
+            />
+            <CWTextInput
+              disabled
+              fullWidth
+              label="Community URL"
+              placeholder="Community URL"
+              value={`${window.location.origin}/${
+                getFormValues?.()?.['communityName'] || community.id
+              }`}
+            />
+            {featureFlags.isCommunityStakesEnabled && (
+              <>
+                <CWTextInput
+                  disabled
+                  fullWidth
+                  label="Community Namespace"
+                  placeholder="Community Namespace"
+                />
+                <CWTextInput
+                  disabled
+                  fullWidth
+                  label="Community Symbol"
+                  placeholder="Community Symbol"
+                  value={community.default_symbol || ''}
+                />
+              </>
+            )}
+            <CWTextArea
+              hookToForm
+              name="communityDescription"
+              label="Community Description"
+              placeholder="Enter a description of your community or project"
+            />
+            <CWCoverImageUploader
+              hookToForm
+              enableGenerativeAI
+              showUploadAndGenerateText
+              name="communityProfileImageURL"
+              canSelectImageBehaviour={false}
+              uploadCompleteCallback={console.log}
+              defaultImageBehaviour={ImageBehavior.Circle}
+              onImageProcessStatusChange={setIsProcessingProfileImage}
+              subheaderText="Community Profile Image (Accepts JPG and PNG files)"
+            />
+          </section>
+
+          <section className="links-section">
+            <div className="header">
+              <CWText type="h4">Links</CWText>
+              <CWText type="b1">
+                Add your Discord, Twitter (X), Telegram, Github, Website, etc.
```

Comment:
I think this is a time we should fix the links. 
1. We should unlock links [in create community flow](https://github.com/hicommonwealth/commonwealth/blob/e8dc6507b8cf32147794ce9656b3f354f552b5f5/packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/BasicInformationForm.tsx#L39) 
2. We should update the icons to new ones
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/82b5847a-f48a-4f01-8005-8089c39f32d5)
3. We should make sure that every link added by the admin is visible in sidebar (or limit number of links that might be added)
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/3e7b188d-1fb5-400c-8452-456af4a968c3)
4. We should basically fix the mess related with social links because we treat them differently on the front end and backend in terms of recognising social platforms, using regexps etc ([source](https://github.com/hicommonwealth/commonwealth/blob/d4d56eb2c6239856deda9ded8a760ebee1aa6755/packages/commonwealth/client/scripts/models/ChainInfo.ts#L326)) 

It also might be another ticket, but would be good to fix it before new admin pages are deployed

Reply:
1. I asked about this form Sachin 

> In designs we have the "[Community links](https://www.figma.com/file/ibuywR0bBcy3ZZhPXbjEJw/Community-Admin?type=design&node-id=278-6592&mode=dev)" form fields added to page. Previously we removed it from the new Create Community Form. Are we going to continue with the new community links design, if yes should we add those back to the create community form as well?

got this reply

> Community links should remain here, the user can add the links from this page. We will be adding it to back to create community later on, not a priority right now.

2. Would love to update icon designs but not sure if they are a priority right now. Will ask from Jared for new designs and if short work, will include in this PR.

3. Agree about this, will add that in this PR.

4. I do agree about this, but I think we should do that as a follow-up (we have a tight schedule for this feature)

Reply:
thanks for research / update